index.html

```diff
@@ -0,0 +1,205 @@
+          <img src="images/works/work3.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul>
+            <li class="worksubtitle">CANOPY</li>
+            <li class="bullets">Back End Dev</li>
+            <li class="bullets">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+
+        <!-- 4rt work! -->
+        <div class="works">
+          <img src="images/works/work4.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul>
+            <li class="worksubtitle">CANOPY</li>
+            <li class="bullets">Back End Dev</li>
+            <li class="bullets">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+      </section>
+
+      <!-- About me section! -->
+      <section class="aboutme">
+        <h1 class="title">About Myself</h1>
+        <p class="txt">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <div class="socialresume">
+          <h3 class="hsocial">LET'S CONNECT</h3>
+          <ul>
+            <li class="social">
+              <a href="#"><img src="images/twitter.svg" alt="" /></a>
+            </li>
+            <li class="social">
+              <a href="#"><img src="images/Linkedin.svg" alt="" /></a>
+            </li>
+            <li class="social">
+              <a href="#"><img src="images/M.svg" alt="" /></a>
+            </li>
+            <li class="social">
+              <a href="#"><img src="images/cat.svg" alt="" /></a>
+            </li>
+            <li class="social">
+              <a href="#"><img src="images/Y.svg" alt="" /></a>
+            </li>
+          </ul>
+          <div class="moreabout"><a href="#">Get my resume</a></div>
+        </div>
+        <div class="resume">
+          <div>
+            <h2 class="subt">Languages</h2>
+            <img src="images/works/Uniona.png" alt="" />
+          </div>
+          <ul>
+            <li><img src="images/works/js.svg" alt="" />javaScript</li>
+            <li><img src="images/works/html.svg" alt="" />HTML</li>
+            <li><img src="images/works/css.svg" alt="" />CSS</li>
+          </ul>
+        </div>
```

Comment:
- Optional: Kindly be sure to follow the Figma design. There is a space in between the languages. Do implement for the next milestone.
- Your design:
<img width="368" alt="Screenshot 2022-03-20 at 1 52 00 AM" src="https://user-images.githubusercontent.com/70269507/159141146-63991739-b2bd-4161-bbb4-4b560fb9d4a8.png">
- Figma design:
<img width="301" alt="Screenshot 2022-03-20 at 1 03 54 AM" src="https://user-images.githubusercontent.com/70269507/159141160-20cfdb6e-ffcc-4f4a-8341-81802c0dd23f.png">